doc/content/reference/application-server-data/webhooks.md

```diff
@@ -47,3 +47,5 @@ $ curl https://thethings.example.com/api/v3/as/applications/app1/webhooks/wh1/de
 ```
 
 Will push a downlink to the end device `dev1` of the application `app1` using the webhook `wh1`.
+
+You can also save the API key in the webhook configuration page using the the **Downlink API Key** field. The Application Server will provide it to your endpoint using the `X-Downlink-APIKey` header and the push and replace operations paths using the `X-Downlink-Push` and `X-Downlink-Replace` headers.
```

Comment:
Is that the right casing for `X-Downlink-APIKey`? I'm seeing `Apikey` in tests below

Reply:
The HTTP standard normalizes the header names to PascalCase, so I'll change it everywhere to `X-Downlink-Apikey`.

Reply:
Thought so, indeed make it consistent